# map parameters
map:
  map_rate: 10
  # in meters, this should be the same as the resolution of the global map
  resolution: 0.2
  horizontal:
    # number of laser lines in horizontal
    laser_line_num: 90
    # in degree, the horizontal angle range
    laser_range_dgr: 90.0
  vertical:
    # number of laser lines in vertical
    laser_line_num: 10
    # in degree, vertical angle range
    laser_range_dgr: 30.0